sample_config.json

```diff
@@ -0,0 +1,222 @@
+[
```

Comment:
Not quite complete yet, but this will be approximately the persisted settings form